tools/RAiDER/models/hrrr.py

```diff
@@ -59,16 +68,21 @@ def download_hrrr_file(ll_bounds, DATE, out, model='hrrr', product='nat', fxx=0,
         raise ValueError
 
     ds_out = None
-
-    for ds in ds_list:
-        if 'isobaricInhPa' in ds._coord_names:
-            ds_out = ds
-            coord = 'isobaricInhPa'
-            break
-        elif 'hybrid' in ds._coord_names:
-            ds_out = ds
-            coord = 'hybrid'
-            break
+    # Note order coord names are request for `test_HRRR_ztd` matters
+    # when both coord names are retreived by Herbie is ds_list possibly in
+    # Different orders on different machines; `hybrid` is what is expected for the test.
+    ds_list_filt_0 = [ds for ds in ds_list if 'hybrid' in ds._coord_names]
+    ds_list_filt_1 = [ds for ds in ds_list if 'isobaricInhPa' in ds._coord_names]
+    if ds_list_filt_0:
+        ds_out = ds_list_filt_0[0]
+        coord = 'hybrid'
+    #  I do not think that this coord name will result in successful processing nominally as variables are
```

Comment:
Change this comment to "TODO: check whether `isobaricInhPa` workflow will result in successfull processing"